src/assets/i18n/en.json5

```diff
@@ -5602,20 +5596,6 @@
 
   "access-control-bitstream-toggle.disable": "Disable option to perform changes on the bitstreams",
 
-  "access-control-mode": "Mode",
-
-  "access-control-access-conditions": "Access conditions",
-
-  "access-control-no-access-conditions-warning-message": "Currently, no access conditions are specified below. If executed, this will replace the current access conditions with the default access conditions inherited from the owning collection.",
-
-  "access-control-replace-all": "Replace access conditions",
-
-  "access-control-add-to-existing": "Add to existing ones",
-
-  "access-control-limit-to-specific": "Limit the changes to specific bitstreams",
-
-  "access-control-process-all-bitstreams": "Update all the bitstreams in the item",
```

Comment:
@GauravD2t : It appears you are still accidently removing i18n keys from `en.json5` which were added by other PRs.  These keys MUST be restored on lines 5605 to 5618 here.